src/components/validations/validations.stories.js

```diff
@@ -179,4 +182,39 @@ storiesOf('Validations', module)
         </Row>
       </Form>
     );
+  })
+  .add('ButtonToggleGroup', () => {
+    const test = value => new Promise((resolve, reject) => {
+      if (value === 'baz') return resolve(true);
+      return reject(Error('Baz is required!'));
+    });
+
+    return (
+      <Form
+        onSubmit={ (ev) => { ev.preventDefault(); } }
```

Comment:
I think would be good to have an action logs here

Reply:
logs added